homeassistant/components/automate/sensor.py

```diff
@@ -0,0 +1,96 @@
+"""Support for Automate Roller Blind Batteries."""
```

Comment:
This PR adds multiple platforms. Please reduce this PR to have 1 platform to start with. Additional platforms can be added in later PRs.

Reply:
To clarify, I should remove the battery level and signal sensors, and put in another PR? (They are part of the same device, and will require some rework to remove from the setup etc...)

Reply:
No, this integration provides the cover and sensor platform. It should contain either of those, not both.